deps.edn

```diff
@@ -153,7 +153,7 @@
   org.eclipse.jetty.websocket/websocket-jetty-server {:mvn/version "11.0.20"}   ; ring-jetty-adapter needs that
   org.flatland/ordered                      {:mvn/version "1.15.11"}            ; ordered maps & sets
   org.graalvm.js/js                         {:mvn/version "22.3.5"}             ; JavaScript engine
-  org.liquibase/liquibase-core              {:mvn/version "4.25.1"              ; migration management (Java lib)
+  org.liquibase/liquibase-core              {:mvn/version "4.21.1"              ; migration management (Java lib)
```

Comment:
This was upped in https://github.com/metabase/metabase/pull/40229. [We think](https://metaboat.slack.com/archives/CKZEMT1MJ/p1710831335780959?thread_ts=1710830406.521509&cid=CKZEMT1MJ) this might cause the [tests to fail](https://github.com/metabase/metabase/actions/runs/8334045632/job/22818627955?pr=40249).